docs/recipes/shared-workers.md

```diff
@@ -34,6 +34,25 @@ const shared = registerSharedWorker({
 
 Within a test process you can only register one worker for each `filename`. Filenames are compared as-is, without normalization. If you call `registerSharedWorker()` a second time, the same worker instance is returned.
 
+If for some reason you want to load the same file as multiple different workers, you can append a unique hash to the end of the filename:
```

Comment:
It may be worth mentioning that `filename` takes a URL, and that's why you can append hashes or even use a query component.